fpt/src/lr35902.rs

```diff
@@ -1653,7 +1662,11 @@ impl LR35902 {
             }
             0xE0 => {
                 // LDH (a8),A
-                self.set_mem8(0xFF00 | self.get_d8(0) as u16, self.a());
+                let mem = 0xFF00 | self.get_d8(0) as u16;
+                if mem as usize == memory::map::BANK && self.a() != 0 {
+                    self.mem.unload_bootrom();
+                }
```

Comment:
you're very right! fixed that. are there any other similar checks?

Reply:
I didn't spot anything in the meantime, but I haven't searched thoroughly.
I wonder if it will make more sense to ditch the `mem: [u8; 65536]` in favor of branching on address ranges in `read*`/`write*` and have all the registers stored separately, because we might have to perform some of these checks anyways, at least in `write*`

Reply:
hm. indeed. lets see if that patterns emerges in the future. it probably will for e.g. audio